src/front/app/components/Create/checkboxes.scss

```diff
@@ -0,0 +1,20 @@
+.challenge-create-enabled__title {
+    padding: 20px 0 0 30px;
+    font-weight: bold;
+}
+
+.challenge-create-disabled__title {
+    padding: 20px 0 0 30px;
+    color: $disabled-point-color;
+    font-family: $checkbox-title-font;
+}
+
+.challenge-create-disabled__checkbox {
```

Comment:
is there some difference between  ?
challenge-create-disabled__checkbox
and
challenge-create-enabled__checkbox